lib/easy_meli/error_parser.rb

```diff
@@ -0,0 +1,38 @@
+class EasyMeli::ErrorParser
+  ERROR_LIST = {
+    'Error validating grant' => EasyMeli::InvalidGrantError,
+    'The User ID must match the consultant\'s' => EasyMeli::ForbiddenError,
+    'invalid_token' => EasyMeli::InvalidTokenError,
+    'Malformed access_token' => EasyMeli::MalformedTokenError,
+    'too_many_requests' => EasyMeli::TooManyRequestsError,
+    'unknown_error' => EasyMeli::UnknownError
+  }
+
+  STATUS_ERRORS = {
```

Comment:
I found that some ML server side errors return an empty body. I've created specific errors that inherit from `EasyMeli:: ServerError` to help with the error handling.